.circleci/config.yml

```diff
@@ -69,6 +62,6 @@ workflows:
 
           matrix:
             parameters:
-              python_version: [ py38, py39, py310 ]
-              django_version: [ ">=3.2,<4.0", ">=4.0,<4.1", ">=4.1,<4.2" ]
+              python_version: [ py310, py312 ]
+              django_version: [ ">=4.2,<4.3", ">=5.1,<5.2" ]
```

Comment:
Everything we care about/where we consume this package should now be running Django 4.2.x. We can drop compatibility testing for everything older than that.
I've also added testing on Django 5.1.x so we're flagging any compatibility issues we are likely to encounter when upgrading to Django 5.2.x after the May 2025 elections.